### PR TITLE
General Fixes

### PR DESCRIPTION
**Pull request by kyteinsky**

### CODE REVIEW

File: .github/workflows/psalm.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+jobs:
+  static-analysis:
+    runs-on: ubuntu-latest
+    strategy:
+      # do not stop on another job's failure
+      fail-fast: false
+      matrix:
+        ocp-version: [ 'dev-master', '28' ]
+        php-version: [ '8.0', '8.1', '8.2' ]
```
Comment (marcelklehr) [inline .github/workflows/psalm.yml:25]: 8.3 is out already as well, I think

File: appinfo/info.xml [hunk 2]
```diff
@@ -44,6 +44,20 @@ Negative:
...
+
+* Transcriptions need to be enabled in the Talk app if you need the calls to be transcribed with any Speech to Text provider (including this app). It can be set using this `occ` command:
+
+```
+occ config:app:set spreed call_recording_transcription --value yes
+```
+
+* This app tends to be heavy on CPU. If it starts to be an issue in your normal workflow, you can limit the number of threads used by Whisper in the "Whisper Speech-To-Text" section in the admin settings
+* The generated transcriptions may vary in accuracy based on the spoken language.
+* Per participant transcription in calls is currently not available but PRs are welcome!
```
Comment (marcelklehr) [inline appinfo/info.xml:60]: Nice! Always good to add more info, to avoid incoming issues :)

File: lib/Command/Transcribe.php [hunk 3]
```diff
@@ -39,7 +42,11 @@ protected function configure() {
 	 */
 	protected function execute(InputInterface $input, OutputInterface $output): int {
 		try {
-			$output->writeln($this->stt->transcribe($input->getArgument('path')));
+			$path = $input->getArgument('path');
+			if (!str_starts_with($path, '/')) {
+				$path = getcwd() . '/' . $path;
```
Comment (marcelklehr) [inline lib/Command/Transcribe.php:47]: ah, nice one! :)

**Review by marcelklehr — APPROVED** [review]
General approval 🫡

File: appinfo/info.xml (hunk 2 above)
Comment (kyteinsky) [reply appinfo/info.xml:60]: it was added to the readme long ago but not updated in the info.xml :)

File: .github/workflows/psalm.yml (hunk 1 above)
Comment (kyteinsky) [reply .github/workflows/psalm.yml:25]: but does NC use 8.3 ?

Comment (marcelklehr) [reply .github/workflows/psalm.yml:25]: 29 does, I think. not sure

Comment (kyteinsky) [reply .github/workflows/psalm.yml:25]: cool, one 8.3 check coming right up!

**Pull request merged**